packages/synapse-react-client/src/theme/DefaultTheme.tsx

```diff
@@ -10,14 +23,53 @@ export const defaultMuiThemeOptions: ThemeOptions = {
   palette: palette,
   components: {
     MuiAlert: {
+      defaultProps: {
+        square: true,
+        iconMapping: {
+          error: <HighlightOffTwoTone />,
+          info: <InfoTwoTone />,
+          success: <CheckCircleTwoTone />,
+          warning: <ErrorTwoTone />,
+        },
+      },
       styleOverrides: {
         root: ({ ownerState, theme }) => ({
-          borderRadius: '0px',
-          fontSize: '16px',
+          alignItems: 'center',
+          fontSize: '14px',
           borderLeft: `10px solid ${
             // The default severity is 'success'.
             theme.palette[ownerState.severity || 'success']['main']
           }`,
+          color: theme.palette.grey[900],
```

Comment:
We also have `theme.palette.text` but I'm not sure if that object provides the value you need

Reply:
Good call! I'll switch to the primary text color (which is the same as grey 900) in the next commit